docs/supported-technologies.asciidoc

```diff
@@ -224,6 +224,24 @@ This section lists all supported asynchronous frameworks.
 |===
 
 
+[float]
+[[supported-messaging-frameworks]]
+=== Messaging frameworks
+When using a messaging framework, sender context is propagated so that receiver events are correlated to the
+same trace.
+
+|===
+|Framework |Supported versions | Description | Since
+
+|JMS
+|1.1, 2.0
+|The agent propagates the context when using `javax.jms.MessageConsumer#send` and `javax.jms.JMSConsumer#send` methods
+through a `Message` property
```

Comment:
describe for which APIs we create spans/transactions

Reply:
Not too long for this table??

Reply:
I'd err towards being long but explicit. @bmorelli25 wdyt?

Reply:
+1 on long but explicit in this case. @eyalkoren if it's too long for the table I can come up with a better way to present the information.